### PR TITLE
feat(website): GA updates

### LINKED ISSUE #4498: GA Website changes
- [ ] EA -> Sign up
- [ ] Link clients in Docs
- [ ] Sign up button in navbar
- [ ] Product -> Download link

### PR DESCRIPTION
**Pull request by jamilbk**
Drafting docs update to capture any user-facing changes for next release

- [x] You'll also want to allow HTTPS/443 to [ghcr.io](http://ghcr.io/)  to allow upgrading the gateway
- [x] Linux client DNS changes
- [x] Double-check Linux binary names

Fixes #4498

### CODE REVIEW

File: website/src/components/RootNavbar/index.tsx [hunk 1]
```diff
@@ -22,12 +22,6 @@ export default function RootNavbar() {
         document.getElementById("product-dropdown-link")
       );
     }
-    if (!docsDropdown) {
-      docsDropdown = new Dropdown(
```
Comment (jamilbk) [inline website/src/components/RootNavbar/index.tsx:26]: Removed the link to legacy docs from the Navbar.

Comment (bmanifold) [reply website/src/components/RootNavbar/index.tsx:26]: Nothing blocking here, just wondering if we provide a link anywhere to the legacy docs?

**Review by bmanifold — APPROVED** [review]
LGTM

File: website/src/components/RootNavbar/index.tsx (hunk 1 above)
Comment (jamilbk) [reply website/src/components/RootNavbar/index.tsx:26]: Not directly from the navbar, but we do in relevant places in the new docs, and they're still being crawled by Algolia and Google.

Just was thinking most people clicking that link now will be 1.0 users and didn't want to confuse them.

Comment (bmanifold) [reply website/src/components/RootNavbar/index.tsx:26]: Yeah, totally makes sense.  Was just personally curious if there was anywhere left on the site that did have a link.

**Pull request merged**